dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/LagomKeys.scala

```diff
@@ -29,7 +30,10 @@ object LagomKeys {
   val PlayService = new LagomKey[Boolean]("playService")
 
   /** The URL of the service */
+  @deprecated("1.5.0", "Use LagomServiceBindings instead")
   val LagomServiceUrl = new LagomKey[String]("lagomServiceUrl")
+
+  val LagomServiceBindings = new LagomKey[Seq[DevServerBinding]]("lagomServiceBindings")
```

Comment:
TIL: This is some internal variable in our maven plugin that's `put` somewhere and `get` somewhere else.